runtime.js

```diff
@@ -540,6 +540,13 @@ const _loadScript = async (file, {files = null, parentUrl = null} = {}) => {
     script = script.replace(r, function() {
       return arguments[1] + replacements[index++] + arguments[3];
     });
+    if (instanceId) {
+      script = script.replace("document.monetization", `window.document.monetization${instanceId}`);
+      script = script.replace("document.monetization.addEventListener", `window.document.monetization${instanceId}.addEventListener`);
```

Comment:
Is this line needed? It looks like a superset of the one above.

Also, we shouldn't change  `document` -> `window.document`.

Reply:
d383152